netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -67,13 +82,36 @@ public TkStart(final Base bse) {
 
     @Override
     public Response act(final Request req) throws IOException {
-        final Inbox inbox = new RqAlias(this.base, req).alias().inbox();
-        final long number = inbox.start();
+        final Href href = new RqHref.Base(req).href();
+        final Iterator<String> token = href.param("token").iterator();
+        String tokenkey = "";
+        final Inbox inbox;
+        long number = 0;
```

Comment:
@prahladyeri We can declare this in line 110 below, where it's assigned to the result of `inbox.start()`. That way we can make it final as well.